File: практика по Go/odnomer.go
package main

import (
	"fmt"
)

func main() {
	var n int
	var sum float64 = 0.0
	var num [100]float64
	fmt.Println("Количество элементов: ")
	fmt.Scan(&n)

	if n <= 0 || n > 100 {
		fmt.Println("кол-во элементов массива долно быть от 1 до 100 !")

		return
	}
	fmt.Println("Введите элементы массива: ")
	for i := 0; i < n; i++ {
		fmt.Scan(&num[i])
		sum += num[i]

	}
	average := sum / float64(n)
	fmt.Println("Среднее значение: ", average)
}

File: практика по C++/odnomer.cpp
#include <iostream>
using namespace std;

int main(){
    setlocale(LC_ALL, "RU");
    int n, i;
    float num[100], sum = 0.0, average;
    cout << "Введите количество элементов массива: ";
    cin >> n;
    while(n > 100 || n <=0){
        cout << "Колво-элементов массива должно быть  от 1 до 100!!!" << endl;
        cout << "Введите повторно кол-во массива!" << endl;
        cin >> n;
    }
    for(i = 0; i < n; ++i){
        cout << i + 1 << "Введите элементы массива: ";
        cin >> num[i];
        sum += num[i];
    }
    average = sum / n;
    cout << "Среднее= " << average;
    return 0;
}
